widgets/secrets/src/CreateModal.tsx

```diff
@@ -83,7 +117,22 @@ export default function CreateModal({ toolbox }: CreateModalProps) {
             .finally(unsetFileLoading);
     };
 
-    const { isHiddenValue, secretKey, secretValue } = inputs;
+    function fetchSecretProviders() {
+        const secretActions = new Stage.Common.Secrets.Actions(toolbox.getManager());
+        secretActions
+            .doGetAllSecretProviders()
+            .then(data => {
+                return setSecretProviders(data.items);
+            })
+            .catch(setMessageAsError);
+    }
+    const secretProvidersArray = secretProviders?.map((item: { name: string }) => ({
```

Comment:
Optional:

Maybe more accurate would be:
```suggestion
    const secretProvidersOptions = secretProviders?.map((item: { name: string }) => ({
```
? So that we see the connection with dropdown options.

Reply:
I've already created secretProviderOptions on L69 https://github.com/cloudify-cosmo/cloudify-stage/pull/2370/files#diff-86dea181f436d9c1a482bb2cc809c0c0e4eb9d1f82c50e7d7e72dbbf19f8a20bR69 so I didn't want to make it too similar (only plural version with `s` added) so I would prefer to leave at it is or change it to something different.

Reply:
Got it, valid argument. In that case we may use `secretProvidersDropdownOptions`.
Anyway - it's still optional.

Reply:
good idea, changed it: https://github.com/cloudify-cosmo/cloudify-stage/pull/2370/commits/b1ee6ee3f5e8f93b905965e5ebc854f9807c2329